packages/app/src/cli/utilities/developer-platform-client/app-management-client.ts

```diff
@@ -357,8 +367,27 @@ export class AppManagementClient implements DeveloperPlatformClient {
     }
   }
 
-  async devStoresForOrg(_orgId: string): Promise<OrganizationStore[]> {
-    return []
+  // we are returning OrganizationStore type here because we want to keep types consistent btwn
+  // partners-client and app-management-client. Since we need transferDisabled and convertableToPartnerTest values
+  // from the Partners OrganizationStore schema, we will return this type for now
+  async devStoresForOrg(orgId: string): Promise<OrganizationStore[]> {
+    const storesResult = await businessPlatformOrganizationsRequest<ListDevStoresQuerySchema>(
+      ListDevStoresQuery,
+      this._businessPlatformToken ?? (await this.businessPlatformToken()),
```

Comment:
this should be just `await this.businessPlatformToken()`, that function already returns `_businessPlatformToken` if it exists.

Reply:
oh yes, sorry I didn't remove this, I was playing around with ways to bypass that `isUnitTest()` check from above 😅